addons/remove-from-profile-rows/addon.json

```diff
@@ -0,0 +1,107 @@
+{
+  "name": "Button for removing items from profile rows",
+  "description": "On rows like \"Favorite Projects\" on your profile, add the option to remove projects/users from the list. For example, this will unfavorite a project when used on the \"Favorite Projects\" row.",
+  "tags": ["community", "profiles"],
+  "enabledByDefault": false,
+  "versionAdded": "1.32.0",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["profiles", "https://scratch.mit.edu/users/*/*"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["profiles", "https://scratch.mit.edu/users/*/*"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Only show on the dedicated pages",
+      "id": "dedicated_pages",
+      "type": "boolean",
+      "default": false
+    },
```

Comment:
If this means only show the button to remove items on full lists and not profile pages, then I don't think this option is necessary. Instead, I think the buttons should always only appear on dedicated pages. Profile pages are more likely used for browsing and not editing.

Reply:
Done with https://github.com/ScratchAddons/ScratchAddons/pull/5960/commits/8fc107c0f36b3b74dff9e769cb11ae1d6d900f6c.